Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -973,6 +1008,24 @@ public protocol PurchasesSwiftType: AnyObject {
 
     #endif
 
+    #if os(iOS) || targetEnvironment(macCatalyst) || VISION_OS
+
+    /**
+     * Displays the first specified store in-app message type to the user if there are any available to be shown.
+     * - Important: This should only be used if you disabled these messages from showing automatically
+     * during SDK configuration using ``Configuration/Builder/with(showStoreMessagesAutomatically:)``
+     * ### Related Symbols
+     * - ``Purchases/showStoreMessages()``
+     * - ``Purchases/showStoreMessages(for:)-79d0o``
+     */
+    @available(iOS 16.0, *)
+    @available(macOS, unavailable)
+    @available(watchOS, unavailable)
+    @available(tvOS, unavailable)
+    func showStoreMessages(for types: Set<StoreMessageType>) async
+
+    #endif
```

Comment:
nitpick but why not put this method alongside the others above? would save an #if

Reply:
This extension is separated since it's for swift only methods. Since this method is for swift only, we need to move it here unfortunately.